### ISSUE TITLE
DEM 2D example is broken

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/GiDInterface#759]
Kratos -> DEM -> 2D -> Circles drop example

Error thrown:

```
Traceback (most recent call last):
  File "MainKratos.py", line 33, in <module>
    DEMAnalysisStageWithFlush(model, project_parameters).Run()
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\KratosMultiphysics\analysis_stage.py", line 50, in Run
    self.RunSolutionLoop()
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\KratosMultiphysics\analysis_stage.py", line 67, in RunSolutionLoop
    is_converged = self._GetSolver().SolveSolutionStep()
  File "E:\PROYECTOS\KratosGiD\kratos.gid\exec\kratos\KratosMultiphysics\DEMApplication\sphere_strategy.py", line 378, in SolveSolutionStep
    (self.cplusplus_strategy).SolveSolutionStep()
RuntimeError: Error: While removing elements and nodes, the number of elements and the number of nodes are not the same in the ModelPart!0

in E:/PROYECTOS/Kratos/applications/DEMApplication/custom_utilities/create_and_destroy.cpp:1110:ParticleCreatorDestructor::DestroyParticles
   E:/PROYECTOS/Kratos/applications/DEMApplication/custom_utilities/create_and_destroy.cpp:1155:ParticleCreatorDestructor::DestroyParticles
   E:/PROYECTOS/Kratos/applications/DEMApplication/custom_utilities/create_and_destroy.cpp:1090:ParticleCreatorDestructor::DestroyParticles
   E:/PROYECTOS/Kratos/applications/DEMApplication/custom_strategies/strategies/explicit_solver_strategy.cpp:505:ExplicitSolverStrategy::SearchDEMOperations
   E:/PROYECTOS/Kratos/applications/DEMApplication/custom_strategies/strategies/explicit_solver_strategy.cpp:459:ExplicitSolverStrategy::SolveSolutionStep

```

[dem_2d_example.gid.zip](https://github.com/KratosMultiphysics/GiDInterface/files/5300093/dem_2d_example.gid.zip)

**Comment by jginternational** [comment]
It works with RBall circle mesher. Waiting for GiD version that fixes the Granular mesher for circles when more than 1 surface is meshed. GID-2197

**Comment by maceligueta** [comment]
@farrufat-cimne any news on this?

**Comment by jginternational** [comment]
It's at @escolano 's rooftop

**Comment by escolano** [comment]
It has been fixed in GiD, but won't be publicly available until a new GiD 15.0x official or 15.1.xd developer is released
